README.md

```diff
@@ -145,6 +145,22 @@ config.json:
 }
 ```
 
+### Azure Storage
+Note, that you must have environment variables `AZURE_STORAGE_ACCOUNT` and `AZURE_STORAGE_ACCESS_KEY`, or `AZURE_STORAGE_CONNECTION_STRING` defined for authentication.
+
+config.json:
+```json
+{
+  "manifestFormat": "importmap",
+  "locations": {
+    "prod": {
+      "azureContainer": "static",
+      "azureBlob": "import-map.js"
```

Comment:
The s3-like URL that we usually define in the root is defined in an environment variable for azure. All we need to know is a storage container and file blob. So realistically with azure it will be difficult to have integ/stage at a different domain unless another level of CDN routing is added. For example, a basic azure storage URL is: https://biocheckcdn.blob.core.windows.net/static/import-map.json

Where `biocheckcdn` is defined in a env access variable and `static` is the container and `import-map.json` is the file blob.

Reply:
Ah I see. Hopefully same CDN domain for Azure is fine for most people and use cases